adapters/dynamoDB_adapters/src/main/java/org/atomhopper/dynamodb/model/PersistedEntry.java

```diff
@@ -0,0 +1,21 @@
+package org.atomhopper.dynamodb.model;
+import com.amazonaws.services.dynamodbv2.datamodeling.*;
+import lombok.Data;
+import java.util.*;
+@Data
+@DynamoDBTable(tableName = "entries")
+public class PersistedEntry {
+
+    @DynamoDBHashKey(attributeName = "entryId")
+    private String entryId;
+    @DynamoDBIndexRangeKey(attributeName = "feed",
+            localSecondaryIndexName = "entryId-feed-index")
```

Comment:
LSI can be appended to the name to specify the type of the index.
entryId-feed-LSI

Reply:
ok.will do the changes